documentation/01-Prerequisites.md

```diff
@@ -6,76 +6,46 @@ These are one-time setup instructions that should be executed prior to following
+  
+  - [Windows](#Windows)
+    - [Windows Requirements](#windows-requirements)
+    - [Clone the repository](#clone-the-repository)
     - [Windows build tools](#windows-build-tools)
       - [Using a local script](#using-a-local-script)
       - [Using Chocolatey](#using-chocolatey)
-      - [Manually](#manually)
-  - [Install Python Dependencies](#install-python-dependencies)
-    - [Linux](#linux-python-install)
-    - [Windows](#windows-python-install)
-  - [Install Docker](#install-docker)
-    - [Linux Docker Install](#linux-docker-install)
+      - [Windows Build Manually](#win-build-manually)
+    - [Windows Pythoon Install](#windows-python-install)
     - [Windows Docker Install](#windows-docker-install)
+      
 
 > Note: Most Windows shell commands here expect `powershell` (or [`pwsh`](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows)).
 
-## Requirements
 
-MLOS currently only supports 64-bit Intel/AMD platforms, though ARM64 support is under development.
+MLOS currently supports 64-bit Intel/AMD platforms, though ARM64 support is under development.
+It supports Windows and Linux environments. Below we provide instructions for each OS.
 
-It supports Windows and Linux environments.
 
-- Windows
+## Linux
 
-  > Portions of MLOS use Docker, which requires a Linux VM.  So support for *one* of the following is required:
-  - [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#update-to-wsl-2) (e.g. Windows 10 build >= 2004, including Pro, Enterprise, *and* Home), *or*
-  - [Hyper-V support](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/) (only Windows 10 Pro/Enterprise, *not* ~~Home~~)
-
-  > Note: WSL2 is advised for ease of setup, integrations with Docker, and more flexible resource utilizations benefits.
+### Linux Requirements
 
-  See the [Install Docker](#install-docker) section for more details.
-
-- Linux
   - Ubuntu 16.04 (xenial), 18.04 (bionic), 20.04 (focal)
 
   > Other distros/versions may work, but are untested.
 
-## Clone the repository
-
-Cross platform
-
-```shell
-git clone https://github.com/microsoft/MLOS.git
-```
-
-> See <https://git-scm.com/book/en/v2/Getting-Started-Installing> for help installing `git`.
-
-## Install build tools
-
-### Docker
-
-To automatically setup a Linux build environment using `docker`, run the following:
-
-```sh
-# Select your target Ubuntu version:
-UbuntuVersion=20.04
-# Build the docker image:
-docker build --build-arg=UbuntuVersion=$UbuntuVersion -t mlos/build:ubuntu-$UbuntuVersion .
-```
-
-> Where `UbuntuVersion` can also be set to another supported version of Ubuntu.
-
-> Tip: you can also pass `--build-arg=http_proxy=http:/some-proxy-caching-host:3128` to help direct `apt` and `pip` to fetch the necessary packages via local caches.
-
-See [02-Build.md](./02-Build.md#docker) for instructions on how to run this image.
 
 ### Linux build tools
 
+#### Linux Build Manually

```

Comment:
This section isn't about building on Linux manually.
How about "Manual Linux Build Environment Setup" or something less verbose like "Manual Setup"